README.md

```diff
@@ -1,8 +1,207 @@
+
+[ {
+  "gmrId" : "GMRD00003I3M",
+  "isUnaccompanied" : true,
+  "updatedDateTime" : "2021-08-31T08:52:04.791Z",
+  "inspectionRequired" : true,
+  "reportToLocations": [
+      {
+        "inspectionTypeId": "1",
+        "locationIds": [
+          "L0029A"
+        ]
+      }
+    ],
+  "trailerRegistrationNums" : [ "AB12 CFR", "GB18 WRD" ],
+  "containerReferenceNums" : [ "ABXY2", "GB456" ],
+  "checkedInCrossing" : {
+    "routeId" : "9",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  }
+},
+  {
+  "gmrId" : "GMRD00003I4M",
+  "isUnaccompanied" : true,
+  "updatedDateTime" : "2021-08-31T08:52:04.791Z",
+  "inspectionRequired" : false,
+  "trailerRegistrationNums" : [ "AB12 CFR", "GB18 WRD" ],
+  "containerReferenceNums" : [ "ABXY2", "GB456" ],
+  "readyToEmbark": true,
+  "checkedInCrossing" : {
+    "routeId" : "9",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  }
+} ]
+```
 
-This is a placeholder README.md for a new repository
+```GET  /reference-data ``` <br/>
+The Carrier can get a reference data at anytime.
+Response Body Example
+```
+* **Code:** 200 <br />
+* **Success**
 
-### License
+{
+  "routes": [
+    {
+      "routeId": "102",
+      "routeEffectiveFrom": "2020-01-01",
+      "routeEffectiveTo": "2020-01-02",
+      "departurePortId": 7014,
+      "routeDirection": "UK_OUTBOUND",
+      "arrivalPortId": 6085,
+      "carrierId": 1
+    },
+    {
+      "routeId": "103",
+      "routeEffectiveFrom": "2020-01-01",
+      "departurePortId": 7054,
+      "routeDirection": "UK_OUTBOUND",
+      "arrivalPortId": 1401,
+      "carrierId": 2
+    }
+  ],
+  "ports": [
+    {
+      "portId": 7050,
+      "portRegion": "Scotland",
+      "portEffectiveFrom": "2000-01-01",
+      "portDescription": "Aberdeen",
+      "chiefPortCode": "",
+      "cdsPortCode": "ABDABDABD",
+      "officeOfTransitCustomsOfficeCode": "UNKNOWN?",
+      "timezoneId": "Europe/London",
+      "portCountryCode": "GB"
+    },
+    {
+      "portId": 1401,
+      "portRegion": "",
+      "portEffectiveFrom": "2000-01-01",
+      "portDescription": "Calais",
+      "chiefPortCode": "",
+      "cdsPortCode": "",
+      "officeOfTransitCustomsOfficeCode": "UNKNOWN?",
+      "timezoneId": "Europe/Paris",
+      "portCountryCode": "FR"
+    }
+  ],
+  "carriers": [
+    {
+      "carrierId": 1,
+      "carrierName": "Stena Line",
+      "countryCode": "SE"
+    },
+    {
+      "carrierId": 2,
+      "carrierName": "DFDS Seaways",
+      "countryCode": "DK"
+    }
+  ],
+  "ruleFailures": [
+    {
+      "ruleId": "013_6",
+      "ruleDescription": "There is a problem with the Entry Summary (ENS) Declaration. Check the declaration details."
+    },
+    {
+      "ruleId": "014_2",
+      "ruleDescription": "The EORI cannot be found. Check and try again."
+    }
+  ]
+}
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+```
```

Comment:
Do we need any licence information in the README? (what does it say in the guides?)